uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Importer/LeaveRequestTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+use CRM_Hrjobcontract_Test_Fabricator_HRJobContract as HRJobContractFabricator;
+use CRM_HRLeaveAndAbsences_Test_Fabricator_AbsenceType as AbsenceTypeFabricator;
+
+/**
+ * Class CRM_HRSampleData_Importer_LeaveRequestTest
+ *
+ * @group headless
+ */
+class CRM_HRSampleData_Importer_LeaveRequestTest extends CRM_HRSampleData_BaseCSVProcessorTest {
+
+  public function setUp() {
+    $this->rows = [];
+    $this->rows[] = $this->importHeadersFixture();
+  }
+
+  public function testProcess() {
+    $contactID = 1;
+
+    // The Leave Request API only returns requests overlapping a Job Contract.
+    // So we need to create a Job Contract first
+    HRJobContractFabricator::fabricate(
+      [ 'contact_id' => $contactID ],
+      [ 'period_start_date' => '2017-01-01' ]
+    );
+
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'title' => 'FooBar'
+    ]);
+
+    $absencePeriod = $this->apiGet('LeaveRequest');
+    $this->assertEmpty($absencePeriod);
+
+    $this->rows[] = [
+      $absenceType->title,
+      $contactID,
+      1,
+      '2017-01-16 00:00:00',
+      1,
+      '2017-01-20 00:00:00',
+      1,
+      'leave',
+      0,
+    ];
+
+    // The importer uses a mapping to cover the contact ids in the csv file
```

Comment:
s/cover/convert

Reply:
this was already fixed in one later commit